cmd/registry/controller/local-tasks.go

```diff
@@ -37,45 +39,46 @@ func (task *ExecCommandTask) String() string {
 }
 
 func (task *ExecCommandTask) Run(ctx context.Context) error {
-	//The monitoring metrics/dashboards are built on top of the format of the log messages here.
-	//Check the metric filters before making  any changes to the format.
-	//Location: registry/deployments/controller/dashboard/*
-	taskDetails := fmt.Sprintf("action={%s} taskID={%s}", task.Action.Command, task.TaskID)
+	// The monitoring metrics/dashboards are built on top of the format of the log messages here.
+	// Check the metric filters before making any changes to the format.
+	// Location: registry/deployments/controller/dashboard/*
+	logger := log.WithFields(log.Fields{
```

Comment:
Just want to call it out here. I am assuming this change works with the dashboard queries that we have.

Reply:
These are the dashboard queries I was looking at to maintain backward compatibility.
```
filter: |-
  resource.type="k8s_container"
  resource.labels.project_id="$REGISTRY_PROJECT_IDENTIFIER"
  resource.labels.location="us-central1-a"
  resource.labels.cluster_name="registry-backend"
  resource.labels.namespace_name="default"
  severity>=DEFAULT
  textPayload=~"(Successful|Failed) Execution:"
labelExtractors:
  action: REGEXP_EXTRACT(textPayload, "action={([^})]*)}")
  status: REGEXP_EXTRACT(textPayload, "(Successful|Failed) Execution:")
  task_id: REGEXP_EXTRACT(textPayload, "taskID={([^}]*)}")
```

```
filter: |-
  resource.type="k8s_container"
  resource.labels.project_id="$REGISTRY_PROJECT_IDENTIFIER"
  resource.labels.location="us-central1-a"
  resource.labels.cluster_name="registry-backend"
  resource.labels.namespace_name="default"
  severity>=DEFAULT
  textPayload=~"Generated [0-9.]* actions"
labelExtractors:
  num_tasks: REGEXP_EXTRACT(textPayload, "Generated ([0-9.]*) actions")
```

Here's some example output.
```
 DEBUG[0000] Generating the list of actions... uid=[ 1c0be4fd ]
 DEBUG[0000] Generated 3 actions. Starting Execution... uid=[ 1c0be4fd ]

 DEBUG[0000] Successful Execution:     action={exec echo hello3} taskID={task2} uid=[ 1c0be4fd ]

 DEBUG[0000] Successful Execution:     action={exec echo hello2} taskID={task1} uid=[ 1c0be4fd ]

 DEBUG[0000] Successful Execution:     action={exec echo hello1} taskID={task0} uid=[ 1c0be4fd ]
```

The previous logger had the same format in the parts matched by the dashboard queries.
```
[ e354e13c ] 2021/09/16 16:13:28 Generating the list of actions...
[ e354e13c ] 2021/09/16 16:13:28 Generated 3 actions. Starting Execution...

[ e354e13c ] 2021/09/16 16:13:28 Successful Execution: action={exec echo hello3} taskID={task2}

[ e354e13c ] 2021/09/16 16:13:28 Successful Execution: action={exec echo hello2} taskID={task1}

[ e354e13c ] 2021/09/16 16:13:28 Successful Execution: action={exec echo hello1} taskID={task0}
```